### PR TITLE
[Android] Updated manual for nnstreamer multi-model app

### PR DESCRIPTION
**Pull request by leemgs**
Fixed issue https://github.com/nnstreamer/nnstreamer/issues/2349.

This commit is to update the existing document in order to re-upload
the nnstreamer multi-model application because the application management
of Google Store is changed.

Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

---

### CODE REVIEW

**Comment by leemgs** [comment]
@again4you , PTAL

File: android/example_app/README.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ Before running .apk file on your Android device, You must copy a SSD model and l
 
 Make directory and copy SSD model and label files into the internal storage of your own Android target device.
 
-You can download these files from [nnsuite testcases repository](https://github.com/nnsuite/testcases/tree/master/DeepLearningModels/tensorflow-lite/ssd_mobilenet_v2_coco).
+You can download these files from [nnstreamer testcases repository](https://github.com/nnstreamer/testcases/tree/master/DeepLearningModels/tensorflow-lite/ssd_mobilenet_v2_coco).
```
Comment (jaeyun-jung) [inline android/example_app/README.md:88]: need to revert (testcases repo is in nnsuite)

File: android/example_app/README.md [hunk 2]
```diff
@@ -28,26 +28,26 @@ We built a example using GStreamer tutorials and camera2 source for Android.
...
 [extfiles.tar.xz](common/jni/extfiles.tar.xz) includes external library such as 'ahc'.
 
-[tensorflow-lite-1.13.tar.xz](https://github.com/nnsuite/nnstreamer-android-resource/blob/master/android_api/ext-files/tensorflow-lite-1.13.tar.xz) includes the libraries and header files of tensorflow-lite.
+[tensorflow-lite-1.13.tar.xz](https://github.com/nnstreamer/nnstreamer-android-resource/blob/master/android_api/ext-files/tensorflow-lite-1.13.tar.xz) includes the libraries and header files of tensorflow-lite.
 
 ```
 $ cd $ANDROID_DEV_ROOT/workspace/nnstreamer-example/android/example_app/common/jni
 $ tar xJf ./extfiles.tar.xz
-$ svn --force export https://github.com/nnsuite/nnstreamer-android-resource/trunk/android_api/ext-files/tensorflow-lite-1.13.tar.xz
+$ curl -O https://raw.githubusercontent.com/nnsuite/nnstreamer-android-resource/master/android_api/ext-files/tensorflow-lite-1.13.tar.xz
```
Comment (jaeyun-jung) [inline android/example_app/README.md:50]: please check the changed file name (tensorflow-lite-1.13.1.tar.xz)

File: android/example_app/README.md [hunk 3]
```diff
@@ -28,26 +28,26 @@ We built a example using GStreamer tutorials and camera2 source for Android.
...
+$ git clone https://github.com/nnstreamer/nnstreamer.git
+$ git clone https://github.com/nnstreamer/nnstreamer-example.git
 ```
 
 Extract external libraries into common directory.
 
 [extfiles.tar.xz](common/jni/extfiles.tar.xz) includes external library such as 'ahc'.
 
-[tensorflow-lite-1.13.tar.xz](https://github.com/nnsuite/nnstreamer-android-resource/blob/master/android_api/ext-files/tensorflow-lite-1.13.tar.xz) includes the libraries and header files of tensorflow-lite.
+[tensorflow-lite-1.13.tar.xz](https://github.com/nnstreamer/nnstreamer-android-resource/blob/master/android_api/ext-files/tensorflow-lite-1.13.tar.xz) includes the libraries and header files of tensorflow-lite.
```
Comment (jaeyun-jung) [inline android/example_app/README.md:45]: https://github.com/nnstreamer/nnstreamer-android-resource/tree/master/external

ext-files > external
tensorflow-lite-1.13 > tensorflow-lite-1.13.1

File: android/example_app/README.md (hunk 1 above)
Comment (myungjoo) [reply android/example_app/README.md:88]: Do you want to migrate testcases.git?

**Comment by leemgs** [comment]
@again4you , PTAL, 

Since @again4you is going to handle  the nnstreamer-example including this task, this PR will be closed soon.

File: android/example_app/README.md (hunk 2 above)
Comment (again4you) [reply android/example_app/README.md:50]: Fixed.

File: android/example_app/README.md (hunk 3 above)
Comment (again4you) [reply android/example_app/README.md:45]: Fixed.

File: android/example_app/README.md (hunk 1 above)
Comment (again4you) [reply android/example_app/README.md:88]: I am going to use AWS link (http://nnsuite.mooo.com/warehouse/nnmodels/) for this.

**Pull request merged**